device/cuda/include/traccc/cuda/utils/make_prefix_sum_buff.hpp

```diff
@@ -28,4 +28,4 @@ vecmem::data::vector_buffer<device::prefix_sum_element_t> make_prefix_sum_buff(
     const std::vector<device::prefix_sum_size_t>& sizes, vecmem::copy& copy,
     const traccc::memory_resource& mr);
 
-}  // namespace traccc::cuda
\ No newline at end of file
+}  // namespace traccc::cuda
```

Comment:
While all these updates in the CUDA and SYCL files are not bad, you shouldn't mix in those changes into this PR.